docs/design/expressions/README.md

```diff
@@ -124,12 +138,18 @@ graph BT
 
     expressionEnd["x;"]
 
-    memberAccess --> parens & braces & unqualifiedName
-    negation --> memberAccess
-    complement --> memberAccess
+    top --> parens & braces & unqualifiedName
+
+    constType --> top
+    pointerType --> constType
```

Comment:
Hm... on further consideration, I think it's important that we allow type operators as subexpressions of `as`, so that `p as const i32*` and `const i32* as type` both work.